apps/designer/app/designer/features/sidebar-left/panels/pages/index.tsx

```diff
@@ -0,0 +1,100 @@
+// @todo: consider moving code from index.tsx to separate files similar to other panels
```

Comment:
so far we have ben using index files to export public api only, they are like a definition of public API, so we don't write anything else but exports in there